internal/provider/provider.go

```diff
@@ -74,27 +74,28 @@ func Provider() *schema.Provider {
 		},
 
 		DataSourcesMap: map[string]*schema.Resource{
-			"vcf_domain":      DataSourceDomain(),
-			"vcf_cluster":     DataSourceCluster(),
-			"vcf_credentials": DataSourceCredentials(),
+			"vcf_cluster":      DataSourceCluster(),
+			"vcf_domain":       DataSourceDomain(),
+			"vcf_credentials":  DataSourceCredentials(),
```

Comment:
Why these changes? It is best to leave refactoring changes to a separate MR, not to pollute the current one with changes/complexity

Reply:
This was cleared out in the rebase and push in 695b133.